includes/import/class-wc-product-csv-importer.php

```diff
@@ -84,7 +84,7 @@ protected function read_file() {
 					$this->raw_data[]                                 = $row;
 					$this->file_positions[ count( $this->raw_data ) ] = ftell( $handle );
 
-					if ( ( $this->params['end_pos'] > 0 && ftell( $handle ) >= $this->params['end_pos'] ) || 0 === --$this->params['lines'] ) {
+					if ( ( $this->params['end_pos'] > 0 && ftell( $handle ) >= $this->params['end_pos'] ) || 0 === $this->params['lines'] ) {
```

Comment:
What is this change for?

Reply:
@mikejolley  It's not related with the current issue. While debugging found it and thought it very simple so maybe fix it with this pull request. Should I change it? 

Regards

Reply:
What does it fix?

Reply:
@mikejolley  oops never mind. Thought it was a typo that's why made that change. Just changed back to the original code.